drivers/spi/spi_ll_stm32.c

```diff
@@ -1293,6 +1386,8 @@ static struct spi_stm32_data spi_stm32_dev_data_##id = {		\
 	SPI_CONTEXT_CS_GPIOS_INITIALIZE(DT_DRV_INST(id), ctx)		\
 };									\
 									\
+PM_DEVICE_DT_INST_DEFINE(index, spi_stm32_pm_action);			\
+									\
 DEVICE_DT_INST_DEFINE(id, &spi_stm32_init, NULL,			\
```

Comment:
```suggestion
DEVICE_DT_INST_DEFINE(id, &spi_stm32_init, PM_DEVICE_DT_INST_GET(id),			\
```

Reply:
These suggestions are needed to fix the build for a custom app for `olimex_lora_stm32wl_devkit` board, which has a few SPI instances.